docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,645 @@
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Use \`maxWidth\` when the content of the page is not center aligned.
+        `}
+        defaultCode={`
+<Flex gap={4} direction="column" flex="grow">
+  <PageHeader
+    title="Settings"
+    maxWidth="50%"
+    primaryAction={
+      <Tooltip text="Additional options" idealDirection="up">
+        <IconButton icon="ellipsis" iconColor="darkGray" />
+      </Tooltip>
+    }
+  />
+  <Box
+    display="flex"
+    marginStart={-3}
+    marginEnd={-3}
+    marginBottom={-3}
+    marginTop={-3}
+    wrap
+    width="80%"
+    direction="column"
+  >
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Heading size="md">Edit profile</Heading>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <TextField
+        label="TextField 4"
+        id="textfield4"
+        onChange={() => {}}
+        placeholder="Placeholder"
+      />
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Box display="flex" wrap marginStart={-3} marginEnd={-3} marginBottom={-3} marginTop={-3}>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 5"
+            id="textfield5"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 6"
+            id="textfield6"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+      </Box>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <SelectList
+        label="SelectList"
+        id="selectlist0"
+        options={[
+          {
+            value: 'belgium',
+            label: 'Belgium',
+          },
+          {
+            value: 'france',
+            label: 'France',
+          },
+          {
+            value: 'usa',
+            label: 'USA',
+          },
+        ]}
+        placeholder="Placeholder"
+        onChange={() => {}}
+      />
+    </Box>
+  </Box>
+</Flex>;
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`Be sure to follow any accessibility guidelines for the components used within PageHeader. The heading within PageHeader will be rendered as a level 1 heading, so ensure there are no other level 1 headings present on the page. `}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`title\`, \`subtext\` and actions within PageHeader.`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      description={`PageHeader can support an optional \`primaryAction\`, which can be a [Button](Button), a [Link](/Link) or an [IconButton](/IconButton) with a [Tooltip](/Tooltip) and optional [Dropdown](/Dropdown).`}
```

Comment:
This sentence isn't clear — consider breaking it into two sentences for readability